### PR TITLE
Network Graph Exploration

### PR DESCRIPTION
**Pull request by WraySmith**
Includes the exploratory analysis for a 3D graph (3D graph worked better than the 2D network graph) based on tsne dimensionality reduction as well as a bare bones test app for generating the 3D graph.

Recommend merging as an exploratory analysis product which is complete. Next steps (for a future PR) will include creating a script to run the tsne analysis and generate an output csv and building the 3D graph into the dashboard.

Also includes updating file structure, updating `README.md`, and adding the `team_reflection2.md` document.

### CODE REVIEW

File: reports/tsne_analysis/3D_app.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import pandas as pd
+import plotly.graph_objs as go
+
+# from dash.dependencies import Input, Output, State
```
Comment (mqharris) [inline reports/tsne_analysis/3D_app.py:7]: commented code

**Comment by mqharris** [comment]
`reports/tsne_analysis/board_games.csv` exists
and it looks like the same data inside the app folder in main

you probably know, just pointing it out

**Comment by mqharris** [comment]
I pulled this branch locally and ran the 3D_app.py successfully

**Comment by WraySmith** [comment]
> `reports/tsne_analysis/board_games.csv` exists
> and it looks like the same data inside the app folder in main
> and then again inside reports/exploratory_data_analysis
> 
> you probably know for clean up, just pointing it out

yup, we should have a chat about how we want to structure the data folder wrt all of the analyses that have occurred

**Review by mqharris — APPROVED** [review]

**Pull request merged**